### DISCUSSION TITLE
Question:  Additional autofill field

### DISCUSSION THREAD

**Discussion by Martzy303** [keepassxreboot/keepassxc#9689] [Q&A]
I have Linux Mint with the KeePassXC-Browser addon for Firefox.
I can't seem to find how to autofill an aditional field beside username and password.
I have a site that has "Code" as the first field, and then are the usual username and passwords fields.
Can I have the "Code" also to be autofilled?
I've tried to edit the entry in the KeepassXC, Edit Entry, Advanced tab, have added an additional attribute, but I don't know what do to with it, how can I make it autofill in the field?

Thank you

**Comment by michaelk83** [comment]
`{S:<ATTRIBUTE_NAME>}`
https://keepassxc.org/docs/KeePassXC_UserGuide#_configure_auto_type_sequences
https://keepassxc.org/docs/KeePassXC_UserGuide#_entry_placeholders

**Comment by Martzy303** [comment]
Have read those before, I have the custom field first and then  user and pass, when I activated the autotype hotkey it was filling only user and pass, that's because I did not have focus on the first field, that was the problem, you need focus on the first field, where the sequence starts.

Thank you
